charts/nri-bundle/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart to deploy New Relic integrations bundled together
 name: nri-bundle
-version: 2.23.0
+version: 2.24.0
```

Comment:
no major bump? just to double ensure the breaking change communication

Reply:
Bumping major here as well